config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -37,9 +37,12 @@ presubmits:
         securityContext:
           privileged: true
         resources:
+          limits:
+            cpu: 4
+            memory: 9000Mi
           requests:
-            memory: "9000Mi"
-            cpu: 7400m
+            cpu: 4
```

Comment:
while kind jobs still have to do a build we should not limit the CPU this low
these jobs are equivilant to the presubmit in resource usage, unlike most CI jobs